### PR TITLE
Fixes build and adds update so we can release a zip version as well

### PR DESCRIPTION
**Pull request by nul800sebastiaan**
This is not an Umbraco package but makes it easier for people not using VS or WebMatrix (so they don't have NuGet).

### CODE REVIEW

**Comment by nul800sebastiaan** [comment]
@Shazwazza This is all tested with a clean checkout, hitting `build.bat` and it works now.
We can update the github release of v.1.0.0 to include a zip download once you think this PR is ok.

Ps. Unignoring the build folder didn't add any other new files to commit in the commit dialog so I think that should be fine.

File: build/Build-Release.ps1 [hunk 1]
```diff
@@ -87,4 +92,10 @@ Write-Output "DEBUGGING: " $CoreNuSpec -OutputDirectory $ReleaseFolder -Version
...
 "NuGet packages also created, so if you want to push them just run:"
-"  nuget push $CoreNuSpec"
\ No newline at end of file
+"  nuget push $CoreNuSpec"
+
+# COPY THE MAANUAL INSTALL README OVER
+Copy-Item "$BuildFolder\ReadmeManual.txt" -Destination $ReleaseFolder
+
+# ZIP UP FOR MANUAL INSTALL
+.\7za.exe a -tzip $ReleaseFolder\UmbracoCms.UmbracoExamine.PDF.$ReleaseVersionNumber$PreReleaseName.zip $ReleaseFolder\UmbracoExamine.PDF.pdb  $ReleaseFolder\UmbracoExamine.PDF.dll $ReleaseFolder\ReadmeManual.txt
```
Comment (Shazwazza) [inline build/Build-Release.ps1]: Do we need to include itextsharp here?

Comment (nul800sebastiaan) [inline build/Build-Release.ps1]: Ah yes, forgot about dependencies, I'll have a look!

**Comment by nul800sebastiaan** [comment]
Alright, those are added now!

**Comment by Shazwazza** [comment]
Ace! I'll pull it in an chuck up a download on the release page and point that issue to it.

**Comment by nul800sebastiaan** [comment]
Cool, also have requests for manual install on the blog here: http://umbraco.com/follow-us/blog-archive/2015/3/31/umbracoexaminepdf-finally-available-on-nuget

**Pull request merged**